### PR TITLE
WIP: Add examples to main stream functions

### PR DESCRIPTION
**Pull request by MrJacz**
**Please describe the changes this PR makes and why it should be merged:**
Add more examples to the docs hence of #1585 

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/client/Client.js [hunk 1]
```diff
@@ -263,6 +263,8 @@ class Client extends BaseClient {
   /**
    * Logs out, terminates the connection to Discord, and destroys the client.
    * @returns {Promise}
+   * @example
```
Comment (KingDGrizzle) [inline src/client/Client.js]: Does this _really_ need an example? :P

Comment (MrJacz) [reply src/client/Client.js]: I mean its a simple function but why not just add it in there

Comment (Dev-Yukine) [reply src/client/Client.js]: it returns a Promise, so please atleast try to handle that promise with a simple console.log() or something

Comment (MrJacz) [reply src/client/Client.js]: @Dev-Yukine Good catch, will update sooner or later

File: src/structures/ClientUser.js [hunk 2]
```diff
@@ -199,6 +199,11 @@ class ClientUser extends Structures.get('User') {
    * Sets the full presence of the client user.
    * @param {PresenceData} data Data for the presence
    * @returns {Promise<Presence>}
+   * @example
+   * // Set presence
+   * client.user.setPresence({ game: { name: "with discord.js" }, status: "idle" })
```
Comment (KingDGrizzle) [inline src/structures/ClientUser.js]: Isn't this supposed to be `activity`?

File: src/structures/ClientUser.js [hunk 3]
```diff
@@ -217,6 +222,11 @@ class ClientUser extends Structures.get('User') {
    * Sets the status of the client user.
    * @param {PresenceStatus} status Status to change to
    * @returns {Promise<Presence>}
+   * @example
+   * // Set status
+   * client.user.setStatus('idle')
+   *   .then(presence => console.log(`New status set! ${presence.status}`))
```
Comment (KingDGrizzle) [inline src/structures/ClientUser.js]: And this should probably say `New status set to ${presence.status}`

File: src/structures/ClientUser.js (hunk 2 above)
Comment (FireController1847) [reply src/structures/ClientUser.js]: Yeah it is

File: src/structures/ClientUser.js [hunk 4]
```diff
@@ -241,6 +256,11 @@ class ClientUser extends Structures.get('User') {
    * Sets/removes the AFK flag for the client user.
    * @param {boolean} afk Whether or not the user is AFK
    * @returns {Promise<Presence>}
+   * @example
+   * // Set afk
+   * client.user.setAFK(true)
+   *   .then(presence => console.log(`I'm now afk`))
```
Comment (Lewdcario) [inline src/structures/ClientUser.js]: Unnecessary usage of literals. Escape the character instead.

File: src/structures/ClientUser.js [hunk 5]
```diff
@@ -255,6 +275,11 @@ class ClientUser extends Structures.get('User') {
    * @param {boolean} [options.everyone=true] Whether to include everyone/here mentions
    * @param {Guild|Snowflake} [options.guild] Limit the search to a specific guild
    * @returns {Promise<Message[]>}
+   * @example
+   * // Fetch mentions
+   * client.user.fetchMentions()
+   *   .then(messages => console.log(`${messages.length} mentions`))
```
Comment (Lewdcario) [inline src/structures/ClientUser.js]: Better wording, please, maybe: `Now fetched ${amount} mentions`

**Comment by MrJacz** [comment]
Closing as I don't have time to finish this

**Pull request closed without merging**